### PR TITLE
[WIPTEST] remove the browser manager globals

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
- [x] remove the globals
- [ ] make the drivers accessible
- [ ] screenshot sanity

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4edceeca3c86be273a4cf2f1c837f25890856b9e:

`cfme/utils/browser.py`:
- :red_circle: Line 2:1: **F401** *'atexit' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/fixtures/soft_assert.py [hunk 1]
```diff
@@ -121,7 +120,7 @@ def handle_assert_artifacts(request, fail_message=None):
         short_tb = full_tb = base64_from_text(fail_message)
 
     try:
-        ss = cfme.utils.browser.browser().get_screenshot_as_base64()
+        ss = appliance.browser.widgettastic.selenium.get_screenshot_as_base64()
```
Comment (izapolsk) [inline cfme/fixtures/soft_assert.py]: typo ?

Comment (RonnyPfannschmidt) [reply cfme/fixtures/soft_assert.py]: yes, there are some more in there

**Comment by dajoRH** [comment]
Lint report for commit edb749ad7eade97774e922df405484a2f90e83d4:

`cfme/utils/browser.py`:
- :red_circle: Line 2:1: **F401** *'atexit' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c76d8204d6b4dae3e6dafb7947871d2fe345a6a2:

`cfme/utils/browser.py`:
- :red_circle: Line 2:1: **F401** *'atexit' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1bbb339e322ea76aa5765ad79b1970e47de5490e:

`cfme/utils/tests/test_simple_locators.py`:
- :red_circle: Line 3:1: **F401** *'cfme.utils.appliance.implementations.ui.ViaUI' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 41815e846bec985b7ddb402f8a10d7c258526511:

`cfme/utils/tests/test_simple_locators.py`:
- :red_circle: Line 3:1: **F401** *'cfme.utils.appliance.implementations.ui.ViaUI' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request closed without merging**